### PR TITLE
More ASF licenses (css + html/jinja)

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: superset/assets/vendor/parallel_coordinates/d3.parcoords.css [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/**
```
Comment (bolkedebruin) [inline superset/assets/vendor/parallel_coordinates/d3.parcoords.css]: Is this one ours or 3rd party?

File: superset/assets/vendor/parallel_coordinates/divgrid.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
+/**
```
Comment (bolkedebruin) [inline superset/assets/vendor/parallel_coordinates/divgrid.js]: Idem

File: superset/assets/vendor/pygments.css [hunk 3]
```diff
@@ -1,3 +1,21 @@
+/**
```
Comment (bolkedebruin) [inline superset/assets/vendor/pygments.css]: Idem

**Comment by mistercrunch** [comment]
Addressed comments.

**Pull request merged**